src/olympia/constants/permissions.py

```diff
@@ -0,0 +1,57 @@
+from collections import namedtuple
+
+
+AclPermission = namedtuple('AclPermission', 'app, action')
+
+# Special null rule.
+NONE = AclPermission('None', 'None')
+

```

Comment:
With https://github.com/mozilla/addons-server/pull/5178#discussion_r112623186 you shouldn't be able to remove it… imho… maybe just document what it's being used for is fine.